can/inc/config.h

```diff
@@ -99,28 +99,31 @@ extern const Field SET_TEMPERATURE_LIMIT;
 extern const Field SET_CURRENT_LIMIT;
 
 
-#define BUS_TEST_REQ_BASE 0x7FE0
+// BUS TEST
+#define BUS_TEST_MSK                0xFF0U

```

Comment:
I see the problem with the old bus test mask, but doesn't this one need to be 0x7FF0 like the bus test base? Otherwise if we have a message with a base like  0x0FF0, it will also pass the mask?

Reply:
We're using 11 bit IDs, so the most if can go up to is `0x7FF`